rust/Cargo.toml

```diff
@@ -70,7 +81,7 @@ private-intra-doc-links = "allow" # We don't publish any of our docs but want to
 [patch.crates-io]
 boringtun = { git = "https://github.com/cloudflare/boringtun", branch = "master" }
 str0m = { git = "https://github.com/firezone/str0m", branch = "main" }
-ip_network_table = { git = "https://github.com/edmonds/ip_network_table", branch = "some-useful-traits" } # For `Debug` and `Clone`
+ip_network_table = { git = "https://github.com/edmonds/ip_network_table", branch = "some-useful-traits" }                             # For `Debug` and `Clone`
```

Comment:
Zed's TOML extension seems very opinionated on how to format it, I'll see if I can align it to our current format

Reply:
Hmm, is there a way to disable that? I am not a fan of formatting changes that we don't enforce in CI.

Reply:
I don't mind the spaces here but I'd like to keep dependencies on a single line.

Reply:
just reverted the whole thing and disabled "format-on-save" for TOML on my end